packages/kyt-cli/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": [
+    "../kyt-core/config/.eslintrc.base.json"
```

Comment:
because this linting is just for when you're locally developing I think it's fine to point to another package. We can certainly revisit publishing the eslint configs. Thoughts @delambo?